src/attributes/classes.js

```diff
@@ -7,16 +7,20 @@ define([
 
 var rclass = /[\t\r\n\f]/g;
 
+function getClass( elem ) {
+	return elem.getAttribute && elem.getAttribute( "class" ) || "";
```

Comment:
Here's the test that fails without the check https://github.com/timmywil/jquery/blob/master/test/unit/attributes.js#L1073